regression/esbmc/github_576_binary/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+input.goto
```

Comment:
Would you mind adding the file that you used to generate this `input.goto`? This way we can try to create more complex tests and also help refactor if we change the enconding.

Reply:
Done.